docs/src/pages/packages/async.md

```diff
@@ -63,6 +63,19 @@ export const fetchList = reatomAsync((ctx) => fetch('...'), 'fetchList').pipe(
 onConnect(fetchList.dataAtom, fetchList)
 ```
 
+### Adding data you've fetched to data you've fetched before
```

Comment:
Please, move it to `/pages/packages/async.md`, the docs/packages folder is autogenerated

Reply:
Done ✅